pyiron_atomistics/atomistics/structure/neighbors.py

```diff
@@ -524,6 +526,58 @@ def _check_width(self, width, pbc):
                 return True
         return False
 
+    def get_spherical_harmonics(self, l, m, cutoff_radius=np.inf, rotation=None):
+        """
+        Args:
+            l (int/numpy.array): Degree of the harmonic (int); must have ``l >= 0``.
+            m (int/numpy.array): Order of the harmonic (int); must have ``|m| <= l``.
+            rotation (numpy.array/list): Rotation to make sure phi does not become nan
+
+        Returns:
+            (numpy.array) spherical harmonic values
```

Comment:
For the same reasons as the other comments, this should document the shape (same as `self.distances` I guess).

Reply:
It should be `(n,)`. And again, I don't know what's the expected notation XD `((natoms,) numpy.array)`?

Reply:
Ja, I think that would work.  It's just important, that using it you know which axes mean what.